lib/index.js

```diff
@@ -23,17 +24,16 @@ module.exports = co.wrap(function * (options) {
   let prodConfig
   if (options.config !== false) {
     const configFilePath = options.config || defaultConfigPath
-    try {
-      fileConfig = require(_.cwd(configFilePath))
+    let resolvedConfigFilePath = _.cwd(configFilePath)
+    if ( fs.existsSync(resolvedConfigFilePath) ||
+         fs.existsSync(resolvedConfigFilePath + '.js') ||
+         fs.existsSync(resolvedConfigFilePath + '.json') ) {
```

Comment:
the `configFilePath` may have extension, should stop adding extension if it already ends with `.js` or `.json`